src/analytics/formTracker/formTracker.ts

```diff
@@ -29,10 +30,28 @@ export class FormTracker extends BaseTracker {
         continue;
       }
 
-      if (
-        (element.type === "radio" || element.type === "checkbox") &&
-        element.checked
-      ) {
+      if (element.type === "checkbox" && element.checked) {
+        // Check if checkbox belongs to a group
+        const checkboxInSameGroup = selectedFields.find(
+          (field) => field.name === element.name,
+        );
+        // New: If the checkbox is part of a group, concatenate the current checkbox's value with the existing value of that group's checkbox
+        if (checkboxInSameGroup) {
+          checkboxInSameGroup.value += `, ${element.value}`;
```

Comment:
Question: Why, here, you're taking the value of element.value and in the else you're taking the value of document.querySelector(`label[for="${element.id}"]`)? What is the difference?